### ISSUE TITLE
sbt test fails because of hardcoded local paths

### ISSUE THREAD

**Issue by FranklinChen** [bearrito/scala-pig#2]
`sbt test` fails at

```
[info]   java.io.FileNotFoundException: File /home/bearrito/Git/scala-pig/src/main/pig/top_queries_input_data.txt does not exist.
```

**Comment by FranklinChen** [comment]
`resources` is a good place to stash local stuff for a self-contained project.

**Comment by bearrito** [comment]
I was going to do that and then attempt something like : val source = Source.fromURL(getClass.getResource("/pig.xml")) but I didn't see a method to get the absolute path, which is what PigUnit expects

**Comment by bearrito** [comment]
Eh, nvm. Looks like its Source.fromfile. 

I'll have it fixed in a bit.

**Comment by bearrito** [comment]
I believe it is fixed as of : 18051972a835c1ccf9da9b7fcb066ad4fd585f98.

Let me know if it works for you and I'll close this down.

**Comment by FranklinChen** [comment]
It works.

On Wed, Apr 17, 2013 at 10:42 PM, bearrito notifications@github.com wrote:

> I believe it is fixed as of : 1805197https://github.com/bearrito/scala-pig/commit/18051972a835c1ccf9da9b7fcb066ad4fd585f98
> .
> 
> Let me know if it works for you and I'll close this down.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/bearrito/scala-pig/issues/2#issuecomment-16550931
> .

## 

Franklin Chen
website/blog: http://franklinchen.com/
